birdhouse/components/scheduler/deploy_data_job.env

```diff
@@ -96,17 +108,8 @@ $BIRDHOUSE_AUTODEPLOY_EXTRA_SCHEDULER_JOBS
 
 fi
 
+)
```

Comment:
I suggest adding a comment here (similar to the opening `(`) because the subshell and nested YAML within multiline `echo ""` is hard to track without indents. Maybe add a comment before the `echo` as well.